lexicon/tests/providers/test_namecom.py

```diff
@@ -0,0 +1,148 @@
+"""Integration tests for Name.com"""
+import json
+from unittest import TestCase
+
+import pytest
+from mock import ANY, Mock, patch
+from requests import HTTPError
+
+from lexicon.config import DictConfigSource
+from lexicon.providers.namecom import provider_parser
+from lexicon.tests.providers.integration_tests import (
+    IntegrationTests, _vcr_integration_test
+)
+
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from integration_tests.IntegrationTests
+class NamecomProviderTests(TestCase, IntegrationTests):
+    """TestCase for Name.com"""
+
+    # I don't think we really need some docstrings here.
+    # pylint: disable=missing-function-docstring
```

Comment:
In fact, most of the time I even put this pylint disable directive at the top of the test module ;)